### ISSUE TITLE
Support PHP 8.1 in Gitonomy

### ISSUE THREAD

**Issue by evs-xsarus** [phpro/grumphp-shim#19]
I have a config with git_blacklist and now, using PHP 8.1, I get this error:

```preg_match($pattern = '/^0+$/', $subject = NULL) phar:///sites/shm/vendor/phpro/grumphp-shim/grumphp.phar/vendor/gitonomy/gitlib/src/Gitonomy/Git/Parser/DiffParser.php:87```

Could Gitonomy be updated?

NB turns out, it isn't linked to git_blacklist , might be more global.

**Comment by evs-xsarus** [comment]
Complete trace:

```
Deprecated: preg_match(): Passing null to parameter #2 ($subject) of type string is deprecated in phar:///sites/shm/vendor/phpro/grumphp-shim/grumphp.phar/vendor/gitonomy/gitlib/src/Gitonomy/Git/Parser/DiffParser.php on line 87

Call Stack:
    0.0006     406976   1. {main}() /sites/shm/vendor/bin/grumphp:0
    0.0007     409056   2. include('/sites/shm/vendor/phpro/grumphp-shim/grumphp') /sites/shm/vendor/bin/grumphp:107
    0.0196    1496552   3. require('phar:///sites/shm/vendor/phpro/grumphp-shim/grumphp.phar/bin/grumphp') /sites/shm/vendor/phpro/grumphp-shim/grumphp:13
    0.0196    1496936   4. _HumbugBoxfabbcd4121df\{closure:phar:///sites/shm/vendor/phpro/grumphp-shim/grumphp.phar/bin/grumphp:8-36}() phar:///sites/shm/vendor/phpro/grumphp-shim/grumphp.phar/bin/grumphp:36
    0.1398    7826480   5. _HumbugBoxfabbcd4121df\Symfony\Component\Console\Application->run($input = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Input\ArgvInput { protected $definition = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 1; private $lastArrayArgument = NULL; private $lastOptionalArgument = NULL; private $options = [...]; private $negations = [...]; private $shortcuts = [...] }; protected $stream = NULL; protected $options = ['skip-success-output' => TRUE]; protected $arguments = ['command' => 'git:pre-commit']; protected $interactive = TRUE; private $tokens = [0 => 'git:pre-commit', 1 => '--skip-success-output']; private $parsed = [] }, $output = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\ConsoleOutput { private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}verbosity = 32; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}formatter = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Formatter\OutputFormatter { private $decorated = TRUE; private $styles = [...]; private $styleStack = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Formatter\OutputFormatterStyleStack { ... } }; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\StreamOutput}stream = resource(2) of type (stream); private $stderr = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\StreamOutput { private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}verbosity = 32; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}formatter = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Formatter\OutputFormatter { ... }; private $stream = resource(3) of type (stream) }; private $consoleSectionOutputs = [] }) phar:///sites/shm/vendor/phpro/grumphp-shim/grumphp.phar/bin/grumphp:35
    0.1454    7829376   6. _HumbugBoxfabbcd4121df\Symfony\Component\Console\Application->doRun($input = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Input\ArgvInput { protected $definition = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 1; private $lastArrayArgument = NULL; private $lastOptionalArgument = NULL; private $options = [...]; private $negations = [...]; private $shortcuts = [...] }; protected $stream = NULL; protected $options = ['skip-success-output' => TRUE]; protected $arguments = ['command' => 'git:pre-commit']; protected $interactive = TRUE; private $tokens = [0 => 'git:pre-commit', 1 => '--skip-success-output']; private $parsed = [] }, $output = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\ConsoleOutput { private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}verbosity = 32; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}formatter = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Formatter\OutputFormatter { private $decorated = TRUE; private $styles = [...]; private $styleStack = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Formatter\OutputFormatterStyleStack { ... } }; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\StreamOutput}stream = resource(2) of type (stream); private $stderr = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\StreamOutput { private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}verbosity = 32; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}formatter = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Formatter\OutputFormatter { ... }; private $stream = resource(3) of type (stream) }; private $consoleSectionOutputs = [] }) phar:///sites/shm/vendor/phpro/grumphp-shim/grumphp.phar/vendor/symfony/console/Application.php:157
    0.1535    8721440   7. _HumbugBoxfabbcd4121df\Symfony\Component\Console\Application->doRunCommand($command = class GrumPHP\Console\Command\Git\PreCommitCommand { private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Command\Command}application = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Application { private $commands = [...]; private $wantHelps = FALSE; private $runningCommand = ...; private $name = 'GrumPHP'; private $version = '1.11.0'; private $commandLoader = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\CommandLoader\ContainerCommandLoader { ... }; private $catchExceptions = TRUE; private $autoExit = TRUE; private $definition = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Input\InputDefinition { ... }; private $helperSet = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Helper\HelperSet { ... }; private $dispatcher = class _HumbugBoxfabbcd4121df\Symfony\Component\EventDispatcher\EventDispatcher { ... }; private $terminal = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Terminal { ... }; private $defaultCommand = 'list'; private $singleCommand = FALSE; private $initialized = TRUE; private $signalRegistry = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\SignalRegistry\SignalRegistry { ... }; private $signalsToDispatchEvent = [...] }; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Command\Command}name = 'git:pre-commit'; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Command\Command}processTitle = NULL; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Command\Command}aliases = []; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Command\Command}definition = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 0; private $lastArrayArgument = NULL; private $lastOptionalArgument = NULL; private $options = [...]; private $negations = [...]; private $shortcuts = [...] }; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Command\Command}hidden = FALSE; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Command\Command}help = ''; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Command\Command}description = 'Executed by the pre-commit hook'; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Command\Command}fullDefinition = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 1; private $lastArrayArgument = NULL; private $lastOptionalArgument = NULL; private $options = [...]; private $negations = [...]; private $shortcuts = [...] }; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Command\Command}ignoreValidationErrors = FALSE; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Command\Command}code = NULL; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Command\Command}synopsis = []; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Command\Command}usages = []; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Command\Command}helperSet = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Helper\HelperSet { private $helpers = [...]; private $command = NULL }; private $testSuites = class GrumPHP\Collection\TestSuiteCollection { private ${_HumbugBoxfabbcd4121df\Doctrine\Common\Collections\ArrayCollection}elements = [...] }; private $stdInFilesLocator = class GrumPHP\Locator\StdInFiles { private $changedFilesLocator = class GrumPHP\Locator\ChangedFiles { ... }; private $listedFiles = class GrumPHP\Locator\ListedFiles { ... } }; private $changedFilesLocator = class GrumPHP\Locator\ChangedFiles { private $repository = class GrumPHP\Git\GitRepository { ... }; private $filesystem = class GrumPHP\Util\Filesystem { ... }; private $paths = class GrumPHP\Util\Paths { ... } }; private $taskRunner = class GrumPHP\Runner\TaskRunner { private $tasks = class GrumPHP\Collection\TasksCollection { ... }; private $middleware = class GrumPHP\Runner\MiddlewareStack { ... } } }, $input = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Input\ArgvInput { protected $definition = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 1; private $lastArrayArgument = NULL; private $lastOptionalArgument = NULL; private $options = [...]; private $negations = [...]; private $shortcuts = [...] }; protected $stream = NULL; protected $options = ['skip-success-output' => TRUE]; protected $arguments = ['command' => 'git:pre-commit']; protected $interactive = TRUE; private $tokens = [0 => 'git:pre-commit', 1 => '--skip-success-output']; private $parsed = [] }, $output = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\ConsoleOutput { private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}verbosity = 32; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}formatter = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Formatter\OutputFormatter { private $decorated = TRUE; private $styles = [...]; private $styleStack = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Formatter\OutputFormatterStyleStack { ... } }; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\StreamOutput}stream = resource(2) of type (stream); private $stderr = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\StreamOutput { private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}verbosity = 32; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}formatter = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Formatter\OutputFormatter { ... }; private $stream = resource(3) of type (stream) }; private $consoleSectionOutputs = [] }) phar:///sites/shm/vendor/phpro/grumphp-shim/grumphp.phar/vendor/symfony/console/Application.php:259
    0.1538    8736640   8. _HumbugBoxfabbcd4121df\Symfony\Component\Console\Command\Command->run($input = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Input\ArgvInput { protected $definition = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 1; private $lastArrayArgument = NULL; private $lastOptionalArgument = NULL; private $options = [...]; private $negations = [...]; private $shortcuts = [...] }; protected $stream = NULL; protected $options = ['skip-success-output' => TRUE]; protected $arguments = ['command' => 'git:pre-commit']; protected $interactive = TRUE; private $tokens = [0 => 'git:pre-commit', 1 => '--skip-success-output']; private $parsed = [] }, $output = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\ConsoleOutput { private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}verbosity = 32; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}formatter = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Formatter\OutputFormatter { private $decorated = TRUE; private $styles = [...]; private $styleStack = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Formatter\OutputFormatterStyleStack { ... } }; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\StreamOutput}stream = resource(2) of type (stream); private $stderr = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\StreamOutput { private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}verbosity = 32; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}formatter = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Formatter\OutputFormatter { ... }; private $stream = resource(3) of type (stream) }; private $consoleSectionOutputs = [] }) phar:///sites/shm/vendor/phpro/grumphp-shim/grumphp.phar/vendor/symfony/console/Application.php:870
    0.1538    8736640   9. GrumPHP\Console\Command\Git\PreCommitCommand->execute($input = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Input\ArgvInput { protected $definition = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 1; private $lastArrayArgument = NULL; private $lastOptionalArgument = NULL; private $options = [...]; private $negations = [...]; private $shortcuts = [...] }; protected $stream = NULL; protected $options = ['skip-success-output' => TRUE]; protected $arguments = ['command' => 'git:pre-commit']; protected $interactive = TRUE; private $tokens = [0 => 'git:pre-commit', 1 => '--skip-success-output']; private $parsed = [] }, $output = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\ConsoleOutput { private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}verbosity = 32; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}formatter = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Formatter\OutputFormatter { private $decorated = TRUE; private $styles = [...]; private $styleStack = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Formatter\OutputFormatterStyleStack { ... } }; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\StreamOutput}stream = resource(2) of type (stream); private $stderr = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\StreamOutput { private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}verbosity = 32; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}formatter = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Formatter\OutputFormatter { ... }; private $stream = resource(3) of type (stream) }; private $consoleSectionOutputs = [] }) phar:///sites/shm/vendor/phpro/grumphp-shim/grumphp.phar/vendor/symfony/console/Command/Command.php:259
    0.1538    8802272  10. GrumPHP\Console\Command\Git\PreCommitCommand->getCommittedFiles($io = class GrumPHP\IO\ConsoleIO { private $input = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Input\ArgvInput { protected $definition = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Input\InputDefinition { ... }; protected $stream = NULL; protected $options = [...]; protected $arguments = [...]; protected $interactive = TRUE; private $tokens = [...]; private $parsed = [...] }; private $output = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\ConsoleOutput { private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}verbosity = 32; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\Output}formatter = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Formatter\OutputFormatter { ... }; private ${_HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\StreamOutput}stream = resource(2) of type (stream); private $stderr = class _HumbugBoxfabbcd4121df\Symfony\Component\Console\Output\StreamOutput { ... }; private $consoleSectionOutputs = [...] }; private $stdin = 'diff --git a/.gitlab-ci.yml b/.gitlab-ci.yml\nindex addc171db1b0ab869b6b0b9df46a529fe7d1613d..64092acc454fb649feaffdd5ff5b11e442617956 100644\n--- a/.gitlab-ci.yml\n+++ b/.gitlab-ci.yml\n@@ -1,5 +1,5 @@\n # Specify the correct PHP version used for your project.\n-image: edbizarro/gitlab-ci-pipeline-php:7.4\n+image: thecodingmachine/php:8.1-v4-cli\n \n # Cache between jobs and branches.\n cache:\n@@ -31,11 +31,6 @@ stages:\n   environment:\n     name: production\n \n-.qaTemplate:\n-  before_script:\n-    - s' }) phar:///sites/shm/vendor/phpro/grumphp-shim/grumphp.phar/src/Console/Command/Git/PreCommitCommand.php:62
    0.1767   24588312  11. GrumPHP\Locator\StdInFiles->locate($stdIn = 'diff --git a/.gitlab-ci.yml b/.gitlab-ci.yml\nindex addc171db1b0ab869b6b0b9df46a529fe7d1613d..64092acc454fb649feaffdd5ff5b11e442617956 100644\n--- a/.gitlab-ci.yml\n+++ b/.gitlab-ci.yml\n@@ -1,5 +1,5 @@\n # Specify the correct PHP version used for your project.\n-image: edbizarro/gitlab-ci-pipeline-php:7.4\n+image: thecodingmachine/php:8.1-v4-cli\n \n # Cache between jobs and branches.\n cache:\n@@ -31,11 +31,6 @@ stages:\n   environment:\n     name: production\n \n-.qaTemplate:\n-  before_script:\n-    - s') phar:///sites/shm/vendor/phpro/grumphp-shim/grumphp.phar/src/Console/Command/Git/PreCommitCommand.php:71
    0.1767   24588344  12. GrumPHP\Locator\ChangedFiles->locateFromRawDiffInput($rawDiff = 'diff --git a/.gitlab-ci.yml b/.gitlab-ci.yml\nindex addc171db1b0ab869b6b0b9df46a529fe7d1613d..64092acc454fb649feaffdd5ff5b11e442617956 100644\n--- a/.gitlab-ci.yml\n+++ b/.gitlab-ci.yml\n@@ -1,5 +1,5 @@\n # Specify the correct PHP version used for your project.\n-image: edbizarro/gitlab-ci-pipeline-php:7.4\n+image: thecodingmachine/php:8.1-v4-cli\n \n # Cache between jobs and branches.\n cache:\n@@ -31,11 +31,6 @@ stages:\n   environment:\n     name: production\n \n-.qaTemplate:\n-  before_script:\n-    - s') phar:///sites/shm/vendor/phpro/grumphp-shim/grumphp.phar/src/Locator/StdInFiles.php:25
    0.1767   24588344  13. GrumPHP\Git\GitRepository->createRawDiff($rawDiff = 'diff --git a/.gitlab-ci.yml b/.gitlab-ci.yml\nindex addc171db1b0ab869b6b0b9df46a529fe7d1613d..64092acc454fb649feaffdd5ff5b11e442617956 100644\n--- a/.gitlab-ci.yml\n+++ b/.gitlab-ci.yml\n@@ -1,5 +1,5 @@\n # Specify the correct PHP version used for your project.\n-image: edbizarro/gitlab-ci-pipeline-php:7.4\n+image: thecodingmachine/php:8.1-v4-cli\n \n # Cache between jobs and branches.\n cache:\n@@ -31,11 +31,6 @@ stages:\n   environment:\n     name: production\n \n-.qaTemplate:\n-  before_script:\n-    - s') phar:///sites/shm/vendor/phpro/grumphp-shim/grumphp.phar/src/Locator/ChangedFiles.php:43
    0.1770   24595864  14. _HumbugBoxfabbcd4121df\Gitonomy\Git\Diff\Diff::parse($rawDiff = 'diff --git a/.gitlab-ci.yml b/.gitlab-ci.yml\nindex addc171db1b0ab869b6b0b9df46a529fe7d1613d..64092acc454fb649feaffdd5ff5b11e442617956 100644\n--- a/.gitlab-ci.yml\n+++ b/.gitlab-ci.yml\n@@ -1,5 +1,5 @@\n # Specify the correct PHP version used for your project.\n-image: edbizarro/gitlab-ci-pipeline-php:7.4\n+image: thecodingmachine/php:8.1-v4-cli\n \n # Cache between jobs and branches.\n cache:\n@@ -31,11 +31,6 @@ stages:\n   environment:\n     name: production\n \n-.qaTemplate:\n-  before_script:\n-    - s') phar:///sites/shm/vendor/phpro/grumphp-shim/grumphp.phar/src/Git/GitRepository.php:59
    0.1775   24631768  15. _HumbugBoxfabbcd4121df\Gitonomy\Git\Parser\ParserBase->parse($content = 'diff --git a/.gitlab-ci.yml b/.gitlab-ci.yml\nindex addc171db1b0ab869b6b0b9df46a529fe7d1613d..64092acc454fb649feaffdd5ff5b11e442617956 100644\n--- a/.gitlab-ci.yml\n+++ b/.gitlab-ci.yml\n@@ -1,5 +1,5 @@\n # Specify the correct PHP version used for your project.\n-image: edbizarro/gitlab-ci-pipeline-php:7.4\n+image: thecodingmachine/php:8.1-v4-cli\n \n # Cache between jobs and branches.\n cache:\n@@ -31,11 +31,6 @@ stages:\n   environment:\n     name: production\n \n-.qaTemplate:\n-  before_script:\n-    - s') phar:///sites/shm/vendor/phpro/grumphp-shim/grumphp.phar/vendor/gitonomy/gitlib/src/Gitonomy/Git/Diff/Diff.php:48
    0.1775   24631768  16. _HumbugBoxfabbcd4121df\Gitonomy\Git\Parser\DiffParser->doParse() phar:///sites/shm/vendor/phpro/grumphp-shim/grumphp.phar/vendor/gitonomy/gitlib/src/Gitonomy/Git/Parser/ParserBase.php:26
    8.6278  149374536  17. preg_match($pattern = '/^0+$/', $subject = NULL) phar:///sites/shm/vendor/phpro/grumphp-shim/grumphp.phar/vendor/gitonomy/gitlib/src/Gitonomy/Git/Parser/DiffParser.php:87

```

**Comment by veewee** [comment]
Thanks for reporting. Built a new version with latest gitonomy

**Comment by evs-xsarus** [comment]
Seems to work :). Tnx.

**Comment by evs-xsarus** [comment]
@veewee I spoke too soon. Which Gitonomy did you use? Seems the bug is fixed in branch 1.3 but not released yet. A 1.3.5 release is required. See commit https://github.com/gitonomy/gitlib/commit/793ffe5826c30e64ea499ea9e4bb353dd0892bef

**Comment by veewee** [comment]
It includes 
https://github.com/gitonomy/gitlib/releases/tag/v1.3.4

Which has this fix:
https://github.com/gitonomy/gitlib/commit/3c2f67a2fa6e7b26dbc9b9d0ca272ed33ba5895e

Looks like that is the wrong fix indeed...

**Comment by evs-xsarus** [comment]
@veewee Gitonomy has released 1.3.5 : https://github.com/gitonomy/gitlib/releases/tag/v1.3.5

**Comment by veewee** [comment]
Sorry for the delay, should be solved now
